providers/src/airflow/providers/google/provider.yaml

```diff
@@ -188,7 +188,7 @@ additional-extras:
       - apache-beam[gcp]
   - name: cncf.kubernetes
     dependencies:
-      - apache-airflow-providers-cncf-kubernetes>=7.2.0
+      - apache-airflow-providers-cncf-kubernetes>10.0.1
```

Comment:
Actually this should be 10.1.0 (and cncf.kuberbetes should also be bumped to 10.1.0). We are effectively working on a "future" version of the cncf.kubernetes provider, which adds new features, it's version will be 10.1.0 - and we should already do it now as part of this change. This the only exception when provider version should be modified by someone else than release manager.

Reply:
Awesome! Bumped the version